splunk/tests/alert.json

```diff
@@ -1,16 +1,5 @@
 {
     "body": {
-        "connection": {
```

Comment:
Add this back to tests

Reply:
Yep, this file is useless without the schema